docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -108,30 +108,30 @@ In the event of a total active region loss, the following data will be lost:
   - **26500** for communication to the Zeebe Gateway from clients/workers.
   - **26501** and **26502** for communication between Zeebe brokers and Zeebe Gateway.
 
-### Zeebe Cluster Configuration
+### Zeebe cluster configuration
 
-Only a combination for Zeebe broker counts and replication factors is supported:
+Only a combination for Zeebe broker counts and replication factors are supported:
```

Comment:
This reads to me as  "Only a combination ~~for Zeebe broker counts and replication factors~~ **is** supported". 

Not sure if it would read better with different phrasing: 

```suggestion
Only the following combinations of Zeebe broker counts and replication factors are supported:
```